test/utils/OrbitalFeatureUtils_tests.jl

```diff
@@ -0,0 +1,7 @@
+using Test
+using DataFrames
+using ..ChemistryFeaturization.Utils.OrbitalFeatureUtils
+
+@testset "OrbitalFeatureUtils" begin
```

Comment:
can you add a couple tests here? I don't think it's necessary for the big "if-tree" ones, but for the slightly less transparently readable functions, it would be nice just to have some example inputs and outputs that live here (personally, I often find tests a nice place to go to figure out answers to questions like "what the heck does this internal function even actually do?")